docker-compose.yml

```diff
@@ -9,11 +9,32 @@ services:
     image: nginx:latest
     volumes:
       - ./src/service-api/conf/nginx.conf:/etc/nginx/nginx.conf
+      - ./nginx-log:/var/log/nginx
     ports:
       - 5000:80
     networks:
       - dps-net
 
+  #############################################################################################
+  ###                  DPS service API SPLUNK UNIVERSAL FORWARDER                           ###
+  #############################################################################################
+  service-api-universalforwarder:
+    image: splunk/universalforwarder:7.3.2
+    volumes:
+      - ./.splunk/conf/inputs.conf:/opt/splunkforwarder/etc/system/local/inputs.conf
+      - ./.splunk/conf/outputs.conf:/opt/splunkforwarder/etc/system/local/outputs.conf
+      - ./nginx-log:/var/log/nginx
+    environment: 
+      - SPLUNK_START_ARGS=--accept-license
+      - SPLUNK_PASSWORD=$SPLUNK_PWD
+      - DPS_NGINX_TCP_HOST=dps-splunk
+      - DPS_NGINX_TCP_PORT=9997
+    ports:
```

Comment:
local dev settings here too

Reply:
Removed since we are not using the env variables